app/models/spree/credit_card.rb

```diff
@@ -118,8 +117,8 @@ def save_requested_by_customer?
 
     private
 
-    def reformat_card_type!(type = nil)
-      self[:cc_type] = active_merchant_card_type(type || cc_type)
+    def reformat_card_type!(type = cc_type)
```

Comment:
Now we actually don't need the default value any more. :slightly_smiling_face: Doesn't really matter.